nucliadb_writer/nucliadb_writer/api/v1/field.py

```diff
@@ -293,16 +294,34 @@ async def add_resource_field_conversation(
 )
 @requires(NucliaDBRoles.WRITER)
 @version(1)
-async def add_resource_field_file(
+async def _add_resource_field_file(
```

Comment:
Using underscore to avoid name collision seems a bit ugly to me. Maybe we can rename the other function `add_resource_field_file_encoded` or something like that and keep this without the underscore?